src/include/txn.h

```diff
@@ -316,43 +345,27 @@ struct __wt_txn {
 
     const char *rollback_reason; /* If rollback, the reason */
 
-/*
- * WT_TXN_HAS_TS_COMMIT --
- *	The transaction has a set commit timestamp.
- * WT_TXN_HAS_TS_DURABLE --
- *	The transaction has an explicitly set durable timestamp (that is, it
- *	hasn't been mirrored from its commit timestamp value).
- * WT_TXN_TS_PUBLISHED --
- *	The transaction has been published to the durable queue. Setting this
- *	flag lets us know that, on release, we need to mark the transaction for
- *	clearing.
- */
-
 /* AUTOMATIC FLAG VALUE GENERATION START */
-#define WT_TXN_AUTOCOMMIT 0x000001u
-#define WT_TXN_ERROR 0x000002u
-#define WT_TXN_HAS_ID 0x000004u
-#define WT_TXN_HAS_SNAPSHOT 0x000008u
-#define WT_TXN_HAS_TS_COMMIT 0x000010u
-#define WT_TXN_HAS_TS_DURABLE 0x000020u
-#define WT_TXN_HAS_TS_PREPARE 0x000040u
-#define WT_TXN_HAS_TS_READ 0x000080u
```

Comment:
Wasn't sure whether to move this one across to the shared flags since we have a `read_timestamp` here too.

Reply:
Yeah, my understanding is the same.